gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    class gaussian_pdf(object):
</pre><p>remove this class and just use function written inside this class directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>